<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClaimInsertDialog</class>
 <widget class="QDialog" name="ClaimInsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="Insert_Date">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-M-d</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>案件编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>理赔状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>911</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>理赔登记</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Commit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>估计损失:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_CostEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Reset">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>450</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_ClaimStatusEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_CarNumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_IncNumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>受理日期:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_CaseNumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>实际理赔金额:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>车牌号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>赔付说明:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_NumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>承保公司编号:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Return">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>450</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>理赔业务编号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_TelEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_PersonEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>被保险人:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>第三者车主责任公司:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_ThirdIncIEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>定损说明:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_LoseEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保险种类:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_StatusEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_DamangeEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_DescriptionEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>241</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>431</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>491</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>519</y>
     <width>911</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>Insert_Date</zorder>
  <zorder>label_4</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>Insert_Commit</zorder>
  <zorder>label_6</zorder>
  <zorder>Insert_CostEdit</zorder>
  <zorder>Insert_Reset</zorder>
  <zorder>Insert_ClaimStatusEdit</zorder>
  <zorder>Insert_CarNumberEdit</zorder>
  <zorder>Insert_IncNumberEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>Insert_CaseNumberEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_9</zorder>
  <zorder>Insert_NumberEdit</zorder>
  <zorder>label_7</zorder>
  <zorder>Insert_Return</zorder>
  <zorder>label_2</zorder>
  <zorder>Insert_TelEdit</zorder>
  <zorder>label_11</zorder>
  <zorder>Insert_PersonEdit</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>Insert_ThirdIncIEdit</zorder>
  <zorder>label_14</zorder>
  <zorder>Insert_LoseEdit</zorder>
  <zorder>label_15</zorder>
  <zorder>Insert_StatusEdit</zorder>
  <zorder>Insert_DamangeEdit</zorder>
  <zorder>Insert_DescriptionEdit</zorder>
  <zorder>label_18</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
